### ISSUE TITLE
[Bug]:  No such contract found for this user_profile

### ISSUE THREAD

**Issue by asifmustafa101** [codeSTACKr/video-source-code-create-nft-collection#117]
### Version

- [X] Yes

### General Question

- [X] No

### What happened?

A bug happened! No such contract found for this user_profile

### What have you tried?

I Googled and asked the community in the discord server but could not find a solution.

### Protect your information

- [X] I Agree

**Comment by reubarm** [comment]
Has anyone found a solution for this?

I have a successful collection response in NFTPort and `npm run get_contract` returns this successfuly, but when doing `npm run mint` it always returns:

`CATCH ERROR: NOK: No such contract found for this user_profile`

**Comment by mrzephros** [comment]
Hi @reubarm, make sure you are on the correct chain and using the correct contract address. You should save two copies of your _deployContractResponse.json one for polygon another for rinkeby. So you can switch around with testnet and mainnet.

Simply call npm run get_contract using the correct _deployContractResponse.json in your build/contract folder to get the correct _contract.json then run your necessary commands

**Comment by reubarm** [comment]
Yes, I was being impatient whilst the contract was indexing lol 😇

Once everything was successfully indexed I used the correct addresses and it worked perfectly!

Thanks for your reply, it's very kind of you to respond 😊

**Comment by mrzephros** [comment]
> Yes, I was being impatient whilst the contract was indexing lol 😇
> 
> Once everything was successfully indexed I used the correct addresses and it worked perfectly!
> 
> Thanks for your reply, it's very kind of you to respond 😊

You are welcome!
